WordPress/Classes/ViewRelated/Reader/ReaderSuggestedTopicsCell.swift

```diff
@@ -0,0 +1,159 @@
+            tableView.trailingAnchor.constraint(equalTo: readableContentGuide.trailingAnchor)
+        ]
+
+        // Constraints for compact horizontal size class
+        compactConstraints = [
+            tableView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor)
+        ]
+
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: cellIdentifier)
+        tableView.isScrollEnabled = false
+        tableView.dataSource = self
+        tableView.delegate = self
+    }
+
+    private func applyStyles() {
+        containerView.backgroundColor = .listForeground
+
+        tableView.backgroundColor = .none
+        tableView.separatorColor = .placeholderElement
+
+        backgroundColor = .none
+
+        refreshHorizontalConstraints()
+    }
+
+    // Activate and deactivate constraints based on horizontal size class
+    private func refreshHorizontalConstraints() {
+        let isCompact = (traitCollection.horizontalSizeClass == .compact)
+
+        compactConstraints.forEach { $0.isActive = isCompact }
+        regularConstraints.forEach { $0.isActive = !isCompact }
+    }
+
+    private enum Constants {
+        static let containerInsets = UIEdgeInsets(top: 8, left: 0, bottom: 0, right: 0)
+    }
+}
+
+extension ReaderSuggestedTopicsCell: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return interests.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier, for: indexPath as IndexPath)
+        cell.textLabel?.text = interests[indexPath.row].title
+        cell.accessoryType = .disclosureIndicator
+        cell.separatorInset = UIEdgeInsets.zero
+        cell.backgroundColor = .none
+        return cell
+    }
+
+    func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+        return UIView(frame: .zero)
+    }
+
+    func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
+        return 1
+    }
+}
+
+extension ReaderSuggestedTopicsCell: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let header = UIView()
+        let headerTitle = UILabel()
+        headerTitle.text = "You might like"
+        header.addSubview(headerTitle)
+        headerTitle.translatesAutoresizingMaskIntoConstraints = false
+        header.pinSubviewToAllEdges(headerTitle, insets: UIEdgeInsets(top: 10, left: 15, bottom: 10, right: 0))
+        headerTitle.font = WPStyleGuide.serifFontForTextStyle(.title2)
+        return header
+    }
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let topic = interests[indexPath.row]
```

Comment:
Isn't it weird / confusing  to use interests and topic instead of just one of them?

Reply:
Yes, I completely agree and that is something that I thought while implementing that.

Since the title says `Topics you might like` and the entity is `ReaderTagTopic` I renamed all to `topics`: e1fbd0161744758df36a12850cb886208d4ef84c